src/components/About/About.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from "react";
+import "./about.css";
+
+const About = () => {
+  return (
+    <div className="about-container">
+      <div className="about-text">
+        <h1 className="main-title">About Us</h1>
+        <h2 className="main-pargraph">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Snsequuntur?
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Snsequuntur?
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Snsequuntur?
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Snsequuntur?
+        </h2>
+
+        <div className="download-btns">
+          <div className="apple-download">
+            <img
+              src="../images/shape.png"
```

Comment:
When using icons or logos we'd prefer to use .svg files instead of png/jpg as they scale without losing sharpness and a couple more benefits.
You can read more on svg's here - https://blog.tbhcreative.com/2017/06/benefits-of-using-svg.html
Some extra reference if you want to know how svgs are made - https://webdesign.tutsplus.com/tutorials/how-to-hand-code-svg--cms-30368

Reply:
Agreed!, thanks!